crates/forge/tests/it/cheats.rs

```diff
@@ -29,6 +30,7 @@ async fn test_cheats_local(test_data: &ForgeTestData) {
 
     let runner = test_data.runner_with(|config| {
         config.fs_permissions = FsPermissions::new(vec![PathPermission::read_write("./")]);
+        config.evm_version = EvmVersion::Prague;
```

Comment:
Is this because foundry-compilers does not support the default Cancun ?

Reply:
it's because resolc doesn't support default prague